openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/FeedRepository.java

```diff
@@ -131,7 +131,7 @@ public Thread create(Thread thread, UUID entityId, EntityReference entityOwner,
     thread.withEntityId(entityId);
 
     // if thread is of type "task", assign a taskId
-    if (thread.getType().equals(ThreadType.Task)) {
+    if (thread.getType() != null && thread.getType().equals(ThreadType.Task)) {
```

Comment:
what cases thread.getType being null?

Reply:
will remove this , I know we have a default threadType = "Conversation", but my subsequent openmetadata-spec build assign threadType = null by default